src/plugin/plugin.ts

```diff
@@ -549,6 +549,36 @@ export interface LogInResult {
   readonly created: boolean;
 }
 
+/**
+ * Holds parameters to initialize the SDK.
+ */
+export interface PurchasesConfiguration {
+  /**
+   * RevenueCat API Key. Needs to be a string
+   */
+  apiKey: string;
+  /**
+   * A unique id for identifying the user
+   */
+  appUserID?: string | null;
+  /**
+   * An optional boolean. Set this to TRUE if you have your own IAP implementation and
+   * want to use only RevenueCat's backend. Default is FALSE. If you are on Android and setting this to ON, you will have
+   * to acknowledge the purchases yourself.
+   */
+  observerMode?: boolean;
+  /**
+   * An optional string. iOS-only, will be ignored for Android. 
+   * Set this if you would like the RevenueCat SDK to store its preferences in a different NSUserDefaults 
+   * suite, otherwise it will use standardUserDefaults. Default is null, which will make the SDK use standardUserDefaults.
+   */
+  userDefaultsSuiteName?: string;
+  /**
+   * An optional boolean. Android only. Required to configure the plugin to be used in the Amazon Appstore.
+   */
+  useAmazon?: boolean;
```

Comment:
Probably out of the scope of this PR but should we add other properties here like whether to use SK2 flag or dangerous settings? We can do that in other PRs too

Reply:
Oh yes! Those properties require work though, so as you say I think it's better to have them as minors